mindsay_sdk/client.py

```diff
@@ -40,7 +41,28 @@ def __init__(self, email: str, production: bool = False):
         self._user_sign_in(email)
         logger.warning('Connected to Mindsay on %s', environment)
 
-    def _user_sign_in(self, email: str):
+    @classmethod
+    def setup_bos_environment(cls, user_email: str, instance_name: Optional[int], language: Optional[str],
```

Comment:
`instance_name` but it's an `int`?

Reply:
Hmm sorry it used to be an int but it's a string now